### PR TITLE
feature/pulsar: add producer option disableBatching

### PR DESCRIPTION
**Pull request by 1046102779**
https://github.com/dapr/components-contrib/pull/1248

### CODE REVIEW

File: pubsub/pulsar/pulsar.go [hunk 1]
```diff
@@ -65,6 +66,15 @@ func parsePulsarMetadata(meta pubsub.Metadata) (*pulsarMetadata, error) {
 		}
 		m.EnableTLS = tls
 	}
+	// DisableBatching is defaultly disable batching.
```
Comment (daixiang0) [inline pubsub/pulsar/pulsar.go:69]: Better to comment as `Set DisableBatching to true as default`.

**Comment by daixiang0** [comment]
I prefer to edit the old one rather than open a new one the next time.

Also please fix DCO issue.

**Comment by 1046102779** [comment]
> I prefer to edit the old one rather than open a new one the next time.
> 
> Also please fix DCO issue.

https://github.com/dapr/components-contrib/pull/1397/checks?check_run_id=4656331073

I follow this guideline, it still fails

**Comment by daixiang0** [comment]
Please use `git rebase upstream/xxx` to sync upstream branch.

**Comment by mukundansundar** [comment]
closed because of #1419

**Pull request closed without merging**